taskt/UI/Forms/Supplement Forms/frmImageCapture.cs

```diff
@@ -1,36 +1,32 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
 using System.Drawing;
 using System.Drawing.Drawing2D;
 using System.Drawing.Imaging;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
 
 namespace taskt.UI.Forms.Supplement_Forms
 {
     public partial class frmImageCapture : Form
     {
+        //These variables control the mouse position
+        private int _selectX;
+        private int _selectY;
+        private int _selectWidth;
+        private int _selectHeight;
+        public Pen SelectPen;
+        public Bitmap UserSelectedBitmap;
```

Comment:
Can you change the ordering here so the public variables are first.

Reply:
done